src/App.js

```diff
@@ -1,24 +1,31 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  NavLink,
+} from 'react-router-dom';
+import AddItem from './components/AddItem';
+import ItemList from './components/ItemList';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Router>
+        <div className="main-content">
+          <Routes>
+            <Route element={<ItemList />} path="/" exact />
+            <Route element={<AddItem />} path="/add-item" />
+          </Routes>
+        </div>
+        <div className="route-links">
+          <NavLink to="/" exact>
+            Item List
+          </NavLink>
```

Comment:
The `exact` prop here is throwing an error in the console. It wasn't immediately clear what was the issue but found that ReactRouter changed the API for `NavLink` between v5 and v6, and we're consuming the latter! Fortunately this is an easy change, just swapping the word `exact` for `end`:
>[This is a simple renaming of a prop to better align with the common practices of other libraries in the React ecosystem.](https://reactrouter.com/docs/en/v6/upgrading/v5#rename-navlink-exact-to-navlink-end)

```suggestion
          <NavLink to="/" end>
            Item List
          </NavLink>
```

Example of error:
![2022-01-14 at 6 06 PM](https://user-images.githubusercontent.com/5881537/149600128-0e826a96-1a4a-413f-88ae-bce10a027d7a.png)

Reply:
Thanks for this explanation! I've now made this change.